example/timer/odp_timer_accuracy.c

```diff
@@ -1072,15 +1072,12 @@ int main(int argc, char *argv[])
 {
 	odp_instance_t instance;
 	odp_init_t init;
-	test_global_t test_global;
+	test_opt_t test_opt;
+	test_global_t *test_global;
 	odp_init_t *init_ptr = NULL;
 	int ret = 0;
 
-	memset(&test_global, 0, sizeof(test_global_t));
-	test_global.stat.nsec_before_min = UINT64_MAX;
-	test_global.stat.nsec_after_min  = UINT64_MAX;
-
-	if (parse_options(argc, argv, &test_global))
+	if (parse_options(argc, argv, &test_opt))
```

Comment:
Test_opt should be memset() to zero to ensure that no member is left uninitialized.

Reply:
That memset is inside parse_options().

Reply:
It is added there in "add cpu count option" commit. It should be added already in this commit ("use shm for global data"), since this changes memory allocation and initialization.

Reply:
Fixed in v4.